e2e/e2e_test.go

```diff
@@ -229,27 +240,21 @@ func verifyTopicMessages(setup *setup.Setup, transactionResponse hedera.Transact
 					ethTransMsgCollected++
 					fmt.Println(fmt.Sprintf("Received Ethereum Transaction Hash [%s]", msg.GetTopicEthTransactionMessage().EthTxHash))
 				}
-
-				// Check whether we collected everything
```

Comment:
I think that leaving the blocking channel is better. We will save execution time and not wait for 60 seconds timeout. In order to make it work we just have to remove the hardcoded 1 so the if becomes:
```go
if expectedValidatorsCount == ethSignaturesCollected &&
 ethTransMsgCollected == expectedEthTxMessageCount {
	c1 <- true
}
```